src/database/migarations/database.ts

```diff
@@ -0,0 +1,8 @@
+import knex from 'knex';
```

Comment:
this folder is used for db migration incase db is down. it should contain .sql file. we can move db connection to utils folder